integrations/applications/google-workspace/resources/template.json

```diff
@@ -0,0 +1,95 @@
+{
+    "payload": {
+        "name": "Google Workspace",
+        "advancedConfigurations": {
+            "discoverableByEndUsers": true,
+            "skipLogoutConsent": true
+        },
+        "imageUrl": "${clientOrigin}/${appBaseNameWithoutTenant}/resources/connections/assets/images/logos/google.svg",
+        "accessUrl": "${serverOrigin}/t/${tenantDomain}/samlsso?spEntityID=${inboundProtocolConfiguration.saml.manualConfiguration.issuer}",
```

Comment:
Are we going to enable the IDP initiated single sign on by default?

Reply:
Let's keep this setting for now.